sphinx/themes/basic/changes/frameset.html

```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Frameset//EN"
   "http://www.w3.org/TR/html4/frameset.dtd">
-<html>
+<html
+  {% if language is not none %}
+  lang = "{{ language }}"
+  {% endif %} >
```

Comment:
It seems the `<html>` tag includes many spaces and newlines.

If the `language` specified:
```
<html xmlns="http://www.w3.org/1999/xhtml"

  lang = "ja"
   >
```

If the `language` is not specified:
```
<html xmlns="http://www.w3.org/1999/xhtml"
   >
```

I'd prefer to more compact HTML.